services/provider/server/server.go

```diff
@@ -792,13 +856,17 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					Kind: "VolumeSnapshotClass",
 					Data: mustMarshal(map[string]string{
 						"csi.storage.k8s.io/snapshotter-secret-name": provisionerSecretName,
-					})},
+					}),
+					Labels: getExternalResourceLabels("VolumeSnapshotClass", replicationEnabled, false, replicationID, storageID),
+				},
 				&pb.ExternalResource{
 					Name: "cephfs",
 					Kind: "VolumeGroupSnapshotClass",
 					Data: mustMarshal(map[string]string{
 						"csi.storage.k8s.io/group-snapshotter-secret-name": provisionerSecretName,
-					})},
+					}),
+					Labels: getExternalResourceLabels("VolumeGroupSnapshotClass", replicationEnabled, false, replicationID, storageID),
```

Comment:
IMO we should use consts for all class kinds, we are passing it as string in multiple places.

Reply:
If you mean replace it in all places not just the function call -- I did that at first, but @nb-ohad opposed to the idea

Reply:
Yes we can easily make mistake when using string in multiple places. we need to have a var or const for it.